concent_api/core/views.py

```diff
@@ -326,6 +327,7 @@ def handle_send_force_get_task_result(request, client_message: message.concents.
     validate_golem_message_task_to_compute(client_message.report_computed_task.task_to_compute)
     validate_id_value(client_message.report_computed_task.task_to_compute.compute_task_def['task_id'],      'task_id')
     validate_id_value(client_message.report_computed_task.task_to_compute.compute_task_def['subtask_id'],   'subtask_id')
+    force_get_task_result_acceptance_time = 2 * settings.CONCENT_MESSAGING_TIME + settings.MAXIMUM_DOWNLOAD_TIME
```

Comment:
Please move this after `if` below, just before this is used for the first time.

Reply:
It's used in `elif` after that `if` so I can move it only if change that `elif` to `if`.